modules/Qnite/plugin/qniteaxis.cpp

```diff
@@ -104,6 +121,22 @@ void QniteAxis::setUpperBound(qreal bound)
     qDebug() << "upper changed" << bound;
     m_upperBound = bound;
     emit upperBoundChanged();
+
+    initTicker();
+  }
+}
+
+bool QniteAxis::flip() const
```

Comment:
Nevermind, what I said is nonsense for properties, `Foo.flip: false ` makes perfectly sense